src/Sarif.Converters/TSLintLoader.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Json;
+using System.IO;
+
+[assembly: InternalsVisibleTo("Sarif.Driver.UnitTests.dll,PublicKey=0024000004800000940000000602000000240000525341310004000001000100433fbf156abe971" +
```

Comment:
In our code base, we conventionally place `InternalsVisibleTo` attributes in the file `<projectDirectory>\Properties\AssemblyInfo.cs`. That file already exists in the `Sarif.Converters` project, and it already grants access to `Sarif.Driver.UnitTests.dll`, so this attribute should not be necessary.

Reply:
The same comment applies to TSLintConverter.cs.

Reply:
I pushed a change to fix this.